cdap-docs/admin-manual/source/upgrading/packages.rst

```diff
@@ -27,9 +27,9 @@ upgrade instructions for the earlier versions and upgrade first to
 
 1. Stop all flows, services, and other programs in all your applications.
 
-#. Stop all CDAP processes::
+#. Stop all CDAP processes (as the user that runs CDAP Master, the CDAP user, indicated by ``<cdap-user>``)::
 
-     $ for i in `ls /etc/init.d/ | grep cdap` ; do sudo service $i stop ; done
+     $ for i in `ls /etc/init.d/ | grep cdap` ; do sudo -u <cdap-user> service $i stop ; done
```

Comment:
perhaps what you want is then; rather than ``sudo``, use ``su``:
```
$ su <cdap-user> -c 'for i in `ls /etc/init.d/ | grep cdap` ; do sudo service $i stop ; done'